src/Workspaces/Core/MSBuild.BuildHost/Rpc/Contracts/TextReaderExtensions.cs

```diff
@@ -19,7 +19,7 @@ internal static class TextReaderExtensions
         // If we're on .NET Core 7.0 the implementation is easy, but on older versions we don't have the helper and since we also don't have
```

Comment:
```suggestion
        // If we're on .NET Core 8.0 the implementation is easy, but on older versions we don't have the helper and since we also don't have
```

Reply:
Should we use `.NET 8.0` instead of `.NET Core 8.0`